src/components/Categories.jsx

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+import { useState, useMemo, useEffect } from 'react';
+
+const Categories = ({setSelectedCategory}) => {
+   
+    const [reviews, setReviews] = useState('');
+    
+    const [loading, setLoading] = useState(true);
+
+    useEffect(()=> {
+      axios
+      .get('https://nialls-games-reviews.herokuapp.com/api/reviews')
+      .then((response) => {
+        setReviews(response.data);
+        setLoading(false);
+      })
+    }, []);
+
+
+    function handleCategoryChange(event) {
+        setSelectedCategory(event.target.value);
+      }
+      
+
+    return (
+    <div>
+        <div className="filter-container">
+          <div>Filter by Category:</div>
+          <div>
+            <select
+              name="category-list"
+              id="category-list"
+              onChange={handleCategoryChange}
+            >
+              <option value="">All</option>
+              <option value="hidden-roles">hidden-roles</option>
+              <option value="dexterity">dexterity</option>
+              <option value="strategy">strategy</option>
+              <option value="deck-building">deck-building</option>
+              <option value="hidden-roles">hidden-roles</option>
```

Comment:
You have hidden roles twice (which wouldn't be able to happen if you were retrieving these from the db.)